tools/SourceKit/CMakeLists.txt

```diff
@@ -500,5 +500,10 @@ include_directories(BEFORE
   ${SOURCEKIT_SOURCE_DIR}/include
 )
 
+if("${CMAKE_SYSTEM_NAME}" STREQUAL "Linux")
```

Comment:
Absolutely. I'm worried about [SR-710](https://bugs.swift.org/browse/SR-710) getting done in time for Swift 3.0, so I'm reducing scope a bit here. Ideally, I'd like SourceKit to run everywhere. Were it able to do so, we'd probably want to change this to handle more than just Linux.